source/scss/_pf-component-alerts.scss

```diff
@@ -0,0 +1,35 @@
+//
+// Alerts
+// --------------------------------------------------
+
+.alert {
+  padding-left: $pf-spacer-xxxxxl;
+  padding-right: $spacer;
+  position: relative;
+
+  .alert-link {
+    color: $link-color;
+    &:hover {
+      color: $link-hover-color;
+    }
+  }
+}
+
+.alert-icon {
+  font-size: $alert-close-font-size;
+  position: absolute;
```

Comment:
Updated to use flexbox.
Needed to add a new <div> around the text, but removed the positions ;)